packages/mobile/src/components/product/product.tsx

```diff
@@ -48,6 +53,7 @@ const Product: FC<Props> = ({ product, contentContainerStyle }) => {
   const { t } = useTranslation();
   const dispatch = useAppDispatch();
   const navigation = useNavigation<NavigationProp<RootNavigationParamList>>();
+  const isListProductUi = productType === ProductDisplayType.LIST;
```

Comment:
This is redundant. You can pass `width` via `contentContainerStyle` prop and button should be removed in ALL lists